src/openloco/objects/competitor_object.h

```diff
@@ -0,0 +1,23 @@
+#pragma once
+
+#include "../localisation/stringmgr.h"
+
+namespace openloco
+{
+#pragma pack(push, 1)
+    struct competitor_object
+    {
+        string_id var_00;        // 0x00
```

Comment:
Var_00 is always the name of the object

Reply:
Have you checked this against locotools for any assistance on naming vars

Reply:
Yeah sorry I wrote another message saying all the information that could be gleamed from LocoTools has been and to ignore my comment but it ended up in a pending review by accident.